test/access_control.sh

```diff
@@ -331,18 +334,11 @@ openssl ca \
     -extensions v3_req \
     -batch \
     -extensions server_ext 2>&1 | artifact_out
-oc rsync -c elasticsearch $certdir $espod:/tmp 2>&1 | artifact_out
-if [ "$espod" != "$esopspod" ] ; then
-    oc rsync -c elasticsearch $certdir $esopspod:/tmp 2>&1 | artifact_out
-fi
```

Comment:
These backups are not useful for debugging any more?

Reply:
not sure what you mean